pom.xml

```diff
@@ -1667,7 +1669,7 @@ under the License.
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-shade-plugin</artifactId>
-					<version>3.0.0</version>
+					<version>3.1.0</version>
```

Comment:
Should we bump it to 3.2.1? see FLINK-13453. And would be great if we also update the document about maven-shade-plugin in [dependencies.md](https://ci.apache.org/projects/flink/flink-docs-master/dev/projectsetup/dependencies.html)